### ISSUE TITLE
Items list resets to top after every update

### ISSUE THREAD

**Issue by arttu-karpinlehto** [bitwarden/clients#5039]
### Steps To Reproduce

It would be nice if the item list did not reset to the first item after every update.

1. Have a long list of items.
2. Scroll the list so that you are not viewing the first page of items.
3. Edit any item and save (deleting an item has the same effect).
4. The item list resets to the top.

Sometimes the list needs editing - for example to prune out duplicate entries. After each edit the list resets to top and scrolling is required to find the place of last edit.

### Expected Result

After editing/deleting, the item list should stay where it was.

### Actual Result

The item list resets to top.

### Operating System

Windows

### Operating System Version

Windows 10 Pro 22H2

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2023.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by MarsCandyBars** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team. I'm only seeing this behavior in the desktop client, I assume that's the client you are using?

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by arttu-karpinlehto** [comment]
Yes indeed, this the Windows desktop client that does this.

**Comment by IDUnavailable** [comment]
Yes, this is annoying (as is not having a multi-select/multi-edit) if you find yourself doing anything that requires updating a bunch of entries. I started sorting my logins into various folders and it's been a moderate annoyance (but probably still easier than doing it via my FF extension or on my phone).

**Comment by inad9300** [comment]
The macOS client suffers from the same problems.

**Comment by elliotwutingfeng** [comment]
I am able to reproduce this on Manjaro Linux x64 (snap).

**Comment by rupak-sinha** [comment]
I came here to post an issue regarding this, just to find this had already been reported.

Same issue in Windows 11 after nearly a year of the actual report.

This is particularly annoying when editing entries in a folder with many entries.

**Comment by pbelbin** [comment]
Yeah, this issue is very annoying.

There's no reason for the list to jump back to the top.

I think bitwarden is great, but, this particular UI bug is very annoying.

Please fix this problem sooner rather than later!

**Comment by dragm83** [comment]
This is still an issue (Windows 2024.9.0)

**Comment by pbelbin** [comment]
yeah, this *is* a very annoying issue

**Comment by djsmith85** [comment]
Closing this as duplicate of https://github.com/bitwarden/clients/issues/2564
